### PR TITLE
need to temporarily set tenant when resolving permissions for user

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/impsvc/auth/DefaultAuthorityService.java [hunk 1]
```diff
@@ -134,14 +138,28 @@ public DefaultAuthorityService(final PermissionService permissionService,
...
+        final String stateId = grants.stream().map(Grant::getStateId).findFirst().orElse(null);
+
+        if (stateId == null) {
+            // to simplify the calling code, return a collection of empty sets
+            orgMap.put(DISTRICT_GROUP, Collections.emptyMap());
+            orgMap.put(DISTRICT, Collections.emptyMap());
+            orgMap.put(INSTITUTION_GROUP, Collections.emptyMap());
+            orgMap.put(INSTITUTION, Collections.emptyMap());
+        } else {
+            TenantContextHolder.setTenantId(stateId);
```
Comment (malaffoon) [inline import-service/src/main/java/org/opentestsystem/rdw/ingest/impsvc/auth/DefaultAuthorityService.java]: This is the change. Not dissimilar to how reporting does it.

File: migrate-olap/src/main/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingConfiguration.java [hunk 2]
```diff
@@ -70,7 +70,7 @@ public Job migrateReportingJob(final JobExecutionListener listener,
                                    final Step finalizeStep) {
         try {
 
-            return jobBuilderFactory.get("Migrate OlAP Reporting Job")
+            return jobBuilderFactory.get("Migrate OLAP Reporting Job")
```
Comment (malaffoon) [inline migrate-olap/src/main/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingConfiguration.java]: unrelated typo that's been driving me nuts for a couple years now

File: import-service/src/main/java/org/opentestsystem/rdw/ingest/impsvc/auth/DefaultAuthorityService.java (hunk 1 above)
Comment (esotrope) [reply import-service/src/main/java/org/opentestsystem/rdw/ingest/impsvc/auth/DefaultAuthorityService.java]: this feels a little awkward. i think these conditions go hand in hand no?:

IF a user has a district|school level grant THEN the user has a state ID in that grant.
If that's the case we shouldn't need an if/else block and we can just always set tenant ID to `grants.stream().map(Grant::getStateId).findFirst().orElse(null);`
I feel like doing that and maybe including a code comment to the effect that the map will be empty when the user has only client-level grants would be more streamline

File: migrate-olap/src/main/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingConfiguration.java (hunk 2 above)
Comment (esotrope) [reply migrate-olap/src/main/java/org/opentestsystem/rdw/ingest/migrate/olap/MigrateOlapReportingConfiguration.java]: haha. the ol'ap

**Pull request merged**